app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+      user = User.find_by(email: params[:email])
+      if user && user.authenticate(params[:password])
+        session[:user_id] = user.id
+        flash[:notice] = "You are now logged in as #{user.first_name}"
+        redirect_to "/profile"
+      else
+        flash[:notice] = "The credentials you have entered are invalid"
+        redirect_to "/login"
+    end
+  end
+
+  def delete
```

Comment:
Would you mind renaming this `destroy`? I think that is the ReSTful convention, and it's what we've done in past projects and seen in demos from instructors.

Reply:
will do

Reply:
Just submitted it, apologies for missing that one